src/platform/osx/Info.plist.in

```diff
@@ -121,6 +121,17 @@
       <string>Alternate</string>
     </dict>
   </array>
+  <key>CFBundleURLTypes</key>
```

Comment:
There's already `CFBundleURLTypes` block above.
You should merge this into that one.

Reply:
Also there's `cmake/modules/MacOSXBundleInfo.plist.in`, which should be also updated.